src/Test.UnitTests.Sarif.WorkItems/SarifWorkItemFilerTests.cs

```diff
@@ -21,7 +22,36 @@ namespace Microsoft.CodeAnalysis.Sarif.WorkItems
     public class SarifWorkItemFilerTests
     {
         [Fact]
-        public void WorkItemFiler_AzureDevOpsClientReceivesExpectedCalls()
+        public void WorkItemFiler_PerRunSplitStrategyRefactorsProperly()
```

Comment:
>Refactors [](start = 53, length = 9)

"Splits" or "Partitions" or "Factors" to avoid the common (in our world) meaning of "refactor". To avoid redundancy in the name, perhaps `PerRunStrategySplitsProperly`.